### ISSUE TITLE
Updating to 1.3.0 causes linker errors

### ISSUE THREAD

**Issue by lmirosevic** [zergtmn/Fabric#22]
Upgrading from `1.2.3` to `1.3.0` has started causing linker errors for me. I am using it as a [dependency](https://github.com/lmirosevic/GBAnalytics/blob/master/GBAnalytics.podspec#L16-17) via [GBAnalytics](https://github.com/lmirosevic/GBAnalytics).

![screen shot 2015-05-28 at 19 17 36](https://cloud.githubusercontent.com/assets/1628832/7866241/7f0e9e06-056e-11e5-8592-773846c8e8c6.png)

**Comment by zergtmn** [comment]
Please check if changing `#import <Fabric/Crashlytics.h>` to `#import <Crashlytics/Crashlytics.h>` makes any difference.

**Comment by lmirosevic** [comment]
Changing it to `#import <Crashlytics/Crashlytics.h>` won't even compile. Perhaps [this](https://github.com/zergtmn/Fabric/commit/715a53a028bddd88bec13f7a2708aefa91f0e456#diff-f6ba1518047c5b2b21164c31e1f0ba5eL20) change to the podspec is causing the problem?

**Comment by joelchen** [comment]
Adding libc++.dylib into your project will fix this issue.

**Comment by zergtmn** [comment]
I believe you don't have to manually add libc++.dylib as a dependency to your project.
Look at the Crashlytics.framework/Modules/module.modulemap:
```
framework module Crashlytics {
    header "Crashlytics.h"
    header "CLSLogging.h"
    header "CLSReport.h"
    header "CLSStackFrame.h"

    export *

    link "z"
    link "c++"
}
```
It should be linked automatically when you import Crashlytics.framework with
`#import <Crashlytics/Crashlytics.h>` or `@import Crashlytics;` or `import Crashlytics` (for Swift).
If Xcode can't find the framework this way there could be an issue with your project's configuration or with the podspec.

**Comment by joelchen** [comment]
Step 1 of your instructions to use Fabric app to configure the components you need (required for this pod to work), dragging the frameworks into the project actually adds libc++.dylib reference along with Fabric and Crashlytics frameworks.

I did not notice libc++.dylib being added a couple of weeks ago from Fabric app though.

Putting that aside, is the process of updating each of the Fabric frameworks in this git repository automated? The official CocoaPods referenced your git repository for Fabric, but the frameworks do not keep up with the latest.

**Comment by sodastsai** [comment]
+1. Add libc++ could fix at least Crashlytics.

**Comment by asidden** [comment]
Had the same issue. It was resolved after I updated my target setting for "Enable Modules (C and Objective-C)" to YES.
![screen shot 2015-06-04 at 3 50 26 pm](https://cloud.githubusercontent.com/assets/226486/7984398/91f2ef5a-0ad1-11e5-9d51-c63c068a5358.png)

**Comment by cakl** [comment]
same issue here. libc++ didn't help and Modules were already enabled.

**Comment by ddebin** [comment]
Same issue here. Enabling Modules solved it. Thanks @asidden  !

**Comment by patoroco** [comment]
@asidden answer solved for me. Should be possible to force that with podspec xcargs or anything else?

**Comment by sebinsua** [comment]
I am getting the same problem but with `Fabric/Twitter`.

```
Undefined symbols for architecture x86_64:
  "_ACAccountTypeIdentifierTwitter", referenced from:
      -[TWTTRAppleSocialAuthenticaticationProvider requestAccessForTwitterAccountsWithCompletion:] in TwitterKit(TWTTRAppleSocialAuthenticaticationProvider.o)
      -[TWTTRAppleSocialAuthenticaticationProvider getTwitterAccounts] in TwitterKit(TWTTRAppleSocialAuthenticaticationProvider.o)
      +[TWTRSystemAccountSerializer saveToSystemAccountCredentials:completion:] in TwitterKit(TWTRSystemAccountSerializer.o)
  "_OBJC_CLASS_$_ACAccount", referenced from:
      objc-class-ref in TwitterKit(TWTRSystemAccountSerializer.o)
  "_OBJC_CLASS_$_ACAccountCredential", referenced from:
      objc-class-ref in TwitterKit(TWTRSystemAccountSerializer.o)
  "_OBJC_CLASS_$_ACAccountStore", referenced from:
      objc-class-ref in TwitterKit(TWTTRAppleSocialAuthenticaticationProvider.o)
      objc-class-ref in TwitterKit(TWTRSystemAccountSerializer.o)
  "_OBJC_CLASS_$_SLComposeViewController", referenced from:
      objc-class-ref in TwitterKit(TWTRComposer.o)
  "_OBJC_CLASS_$_SLRequest", referenced from:
      objc-class-ref in TwitterKit(TWTTRAppleSocialAuthenticaticationProvider.o)
  "_SLServiceTypeTwitter", referenced from:
      -[TWTTRAppleSocialAuthenticaticationProvider performTwitterRequestWithURL:parameters:account:completion:] in TwitterKit(TWTTRAppleSocialAuthenticaticationProvider.o)
      -[TWTRComposer init] in TwitterKit(TWTRComposer.o)
ld: symbol(s) not found for architecture x86_64
```

Does anybody know how to fix? I've tried both solutions mentioned within this thread.

**Comment by lmirosevic** [comment]
Adding the link step for libC++ to `GBAnalytics` solved the issue: https://github.com/lmirosevic/GBAnalytics/commit/23a44d3fc4a36e0cc23d96e493d543a1dfbca99e#diff-790a49cf537dd5c4c0dc139f7833a327R27

**Comment by Dids** [comment]
Enabling Modules did NOT fix this for me (using 1.3.2 of Fabric/Crashlytics).

What did fix it, was manually adding both libstdc++.dylic AND libz.dylib. Issues like these really need to be fixed quickly. :/
